docs/book/driver-deployment/installation.md

```diff
@@ -0,0 +1,226 @@
+# vSphere CSI Driver - Installation
+
+The following steps need to be performed on the k8s node where the vSphere CSI drivers controller will be deployed. We recommend installing vSphere CSI driver on your k8s master node.
```

Comment:
Put a note in the beginning of page that these steps are applicable to Vanila Kubernetes Cluster only.

Reply:
done

Reply:
done